authz/providers/azure/azure.go

```diff
@@ -81,7 +81,7 @@ func (s Authorizer) Check(request *authzv1.SubjectAccessReviewSpec, store authz.
 	}
 
 	// check if user is system accounts
-	if strings.HasPrefix(strings.ToLower(request.User), "system") {
+	if strings.HasPrefix(strings.ToLower(request.User), "system:") {
```

Comment:
any unit test?

Reply:
I need to add unit tests for all skip scenarios in Azure_test.go. Will consider all together as part of that change.